praw/models/reddit/subreddit.py

```diff
@@ -304,6 +420,29 @@ def unread(self, **generator_kwargs):
             self.subreddit._reddit, API_PATH['moderator_unread'].format(
                 subreddit=self.subreddit), **generator_kwargs)
 
+    def update_settings(self, **kwargs):
+        """Update the subreddit's settings.
+
+        See :meth:`Subreddit._create_or_update` for details on each parameter.
```

Comment:
With this modification (https://github.com/praw-dev/praw/pull/642/commits/ca073d17707e4bec41504b6ac167184475f0913b#r72890863) it would be good to add a statement to the end of the docstring to say something to the effect of "additional keyword arguments can be passed to handle new settings as Reddit creates them".